docs/en/connector-v2/sink/Doris.md

```diff
@@ -75,11 +75,9 @@ Using as a multiplier for generating the next delay for backoff
 
 The amount of time to wait before attempting to retry a request to `Doris`
 
-### sink.properties.*  [doris stream load config]
+### doris.config [map]
```

Comment:
Can you add more instructions about this option? For example where can users find these parameters?

Reply:
Thanks，I added the stream load official doc link.